CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Changed
```

Comment:
Without an option to control this with a default of false this becomes a Breaking Change which I am fine with but we should call it out as such per: https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#breaking-changes or add the option.